Common/Symbol.cs

```diff
@@ -454,6 +483,7 @@ public bool Equals(Symbol other)
             SecurityIdentifier sid;
             if (SecurityIdentifier.TryParse(ticker, out sid))
             {
+                Logging.Log.Trace($"Lifted ticker {ticker} - {sid.Symbol}");
```

Comment:
Is this a log we want for production? Thinking it might get verbose - nit: also it's not following current Lean logging conventions, missing scope, maybe `Symbol.ImplicitStringToSymbol()`?

Reply:
Good spot -- I'm gussing not.